src/translations/es.ts

```diff
@@ -0,0 +1,182 @@
+import type { translationDict } from ".";
+
+const dictionary: translationDict = {
+    description: "{title}, Gráfico Sonificado",
+    updated: "{title} Actualizado",
+    "updated-untitled": "Gráfico Actualizado",
+    missing: "Falta", // Faltante is better depending on context, where do we use this?
```

Comment:
This relates to missing data. The data points could be out of range, or perhaps not provided. For example, you might have a bar chart with bars like "Saturday, $100" and "Sunday, missing".